src/toplevelprops.ts

```diff
@@ -58,7 +57,7 @@ export interface AutoSizeParams {
 }
 
 function _normalizeAutoSize(autosize: AutosizeType | AutoSizeParams) {
-  return isString(autosize) ? {type: autosize} : autosize || {};
+  return typeof autosize === 'string' ? {type: autosize} : autosize || {};
```

Comment:
Let's use isString in our local util instead?

Reply:
Done, and rebased.